python/perspective/perspective/table/view.py

```diff
@@ -26,19 +26,25 @@ def __init__(self, Table, config=None):
         View objects are immutable, and will remain in memory and actively process
         updates until its delete() method is called.
         '''
+        self._name = str(random())
```

Comment:
Ideally these names are unique-ish across processes since we're multi-process now - and indeed a browser can have multiple `<perspective-viewer>` bound to different engines in different languages.  Deterministic ID generation is worse for this case IMO.